lms/templates/dashboard.html

```diff
@@ -61,13 +61,9 @@
     });
   </%static:require_module>
   <%static:require_module module_name="js/learner_dashboard/entitlement_unenrollment_factory" class_name="EntitlementUnenrollmentFactory">
-    ## Wait until the document is fully loaded before initializing the EntitlementUnenrollmentView
-    ## to ensure events are setup correctly.
-    $(document).ready(function() {
-      EntitlementUnenrollmentFactory({
-        dashboardPath: "${reverse('dashboard') | n, js_escaped_string}",
-        signInPath: "${reverse('signin_user') | n, js_escaped_string}"
-      });
```

Comment:
This should not be removed. This looks like an issue from resolving a merge conflict?

Reply:
hmm, yeah I definitely didn't explicitly remove it; I will add it back.